Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControlTypes.cs

```diff
@@ -35,7 +35,19 @@ public enum HandleType
     {
         None = 0,
         Rotation,
-        Scale
+        Scale,
+        Translation
+    }
+
+    /// <summary>
+    /// Flags which determine which handle type is enabled.
+    /// </summary>
+    [System.Flags]
+    public enum HandleFlags
+    {
```

Comment:
I *think* we might also want None here to avoid problems like we've seen:

https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8190

Reply:
HandleFlags are no more!